region/lowernorfair/east/The Worst Room In The Game.json

```diff
@@ -851,6 +863,18 @@
         {"heatFrames": 160}
       ]
     },
+    {
+      "link": [5, 3],
+      "name": "HitBox",
+      "requires": [
+        "canHitbox",
+        "h_canUsePowerBombs",
+        {"heatFrames": 180}
+      ],
+      "unlocksDoors": [
+        {"types": ["powerbomb"], "requires": []}
```

Comment:
```suggestion
        {"types": ["powerbomb"], "requires": [], "useImplicitRequires": false}
```
This PR looks fine to merge. There are some strats that could benefit from things like this, which means that the PB from the strat is enough to open the door, so the PB to open the door is not needed.

Reply:
Ah I didn't know about `useImplicitRequires`

Reply:
Another thing I didn't do is let nodes "remote unlock" doors that you aren't directly travelling to next.  Like missiles strats if there is a junction between two doors, or PBs in ScrewAttackRoom.

Reply:
Doing things like that, or opening multiple doors (or blocks or enemies + doors) with a single PB, etc. can get to be a crazy amount of work really fast.